eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp

```diff
@@ -29,6 +29,12 @@ double vespalib_eval_approx(double a, double b) { return (vespalib::approx_equal
 double vespalib_eval_relu(double a) { return std::max(a, 0.0); }
 double vespalib_eval_sigmoid(double a) { return 1.0 / (1.0 + std::exp(-1.0 * a)); }
 double vespalib_eval_elu(double a) { return (a < 0) ? std::exp(a) - 1.0 : a; }
+double vespalib_eval_bit(double a, double b) {
+    // must match Bit::f
```

Comment:
it would be easier to just call Bit::f here

Reply:
I considered it, but did not think gcc would be able to inline it (feels bad to get 2 function calls per bit). Maybe we should have a common inline implementation that we call from both places.